ols-2.md

```diff
@@ -72,9 +73,10 @@ and international bioinformatics communities.
 The vision of Open Life Science program is to strengthen Open Science skills for early stage researchers and young leaders in life science.
 
 At the end of the program, our participants will be able to:
-- Describe and define the terms *openness*, *open science*, *open leadership*, *community interactions*, *value exchanges*, *inclusivity*, *accessibility*, *open Science practices in developing resources and training*
+
+- Describe and define the terms _openness_, _open science_, _open leadership_, _community interactions_, _value exchanges_, _inclusivity_, _accessibility_, _open Science practices in developing resources and training_
 - Learn how to apply those principles to open leadership and working open in their projects and communities
-. Learn how to collect, invite, and tell stories that demonstrate how and why openness benefits the communities they serve
+  . Learn how to collect, invite, and tell stories that demonstrate how and why openness benefits the communities they serve
```

Comment:
Why adding a space here?

Reply:
Not intentional. The editor on my pc did that. I'm sorry